pipelines/backend_tests.go

```diff
@@ -2,21 +2,49 @@ package pipelines
 
 import (
 	"context"
-
 	"dagger.io/dagger"
+	"fmt"
 	"github.com/grafana/grafana-build/containers"
+	"log"
+	"strings"
```

Comment:
This file needs to be formatted with `goimports` instead of `go fmt`. https://pkg.go.dev/golang.org/x/tools/cmd/goimports

It's a superset of the Go formatter which also formats external imports to be grouped and below standard library imports.